modules/grpc/bigquery/bigquery-worker.cpp

```diff
@@ -68,35 +68,6 @@ DestinationWorker::~DestinationWorker()
 bool
 DestinationWorker::init()
 {
-  DestinationDriver *owner = this->get_owner();
-
-  ::grpc::ChannelArguments args{};
-
-  if (owner->keepalive_time != -1)
-    args.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, owner->keepalive_time);
-  if (owner->keepalive_timeout != -1)
-    args.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, owner->keepalive_timeout);
-  if (owner->keepalive_max_pings_without_data != -1)
-    args.SetInt(GRPC_ARG_HTTP2_MAX_PINGS_WITHOUT_DATA, owner->keepalive_max_pings_without_data);
-
-  args.SetInt(GRPC_ARG_KEEPALIVE_PERMIT_WITHOUT_CALLS, 1);
-
-  auto credentials = ::grpc::GoogleDefaultCredentials();
-  if (!credentials)
-    {
-      msg_error("Error querying BigQuery credentials", log_pipe_location_tag((LogPipe *) this->super->super.owner));
-      return false;
-    }
-
-  this->channel = ::grpc::CreateCustomChannel(owner->get_url(), credentials, args);
-  if (!this->channel)
-    {
-      msg_error("Error creating BigQuery gRPC channel", log_pipe_location_tag((LogPipe *) this->super->super.owner));
-      return false;
-    }
-
-  this->stub = google::cloud::bigquery::storage::v1::BigQueryWrite().NewStub(channel);
-
   return log_threaded_dest_worker_init_method(&this->super->super);
```

Comment:
You could probably get rid of this overridden function completely.

Reply:
Thanks, fixed.